cli.js

```diff
@@ -83,7 +83,7 @@ function init(data) {
 
 	const fn = dotProp.get(chalk, styles.join('.'));
 	process.stdout.write(fn(data.replace(/\n$/, '')));
-	if (!cli.flags.noNewline) {
+	if (!cli.flags.noNewline && cli.flags.newline !== false) {
```

Comment:
This could use a short code comment explaining the logic and linking to the `yarns-parser` issue.

Reply:
Ah okay, but I didn't create a `yargs-parser` issue yet because I found a possible way to do it using an existing [`boolean-negation` feature in yargs-parser](https://github.com/yargs/yargs-parser#boolean-negation).

See https://github.com/chalk/chalk-cli/pull/16#issuecomment-918528286

That approach wouldn't require changing `yargs-parser` and would only require a small change to `meow`.

I thought that might be a pretty good approach, since it seemed to require only a small change to meow and since meow is one of your projects, it would be easy to get done.

Check out https://github.com/chalk/chalk-cli/pull/16#issuecomment-918528286 and see if you like that, or if you think it's better to request a feature from `yargs-parser`.

Reply:
@sindresorhus: See 5e680e1. It adds a comment that explains and points the reader to https://github.com/chalk/chalk-cli/issues/30

We can discuss things in that issue and/or link to any meow or yargs-parser issues or PRs that are created.

Reply:
I would prefer an option in `yarns-parser`. That would benefit all consumers using `yarns-parsers` and not just `meow`. It seems like it is something that would be generally useful.

Reply:
Ok. Would you please copy this comment to https://github.com/chalk/chalk-cli/issues/30 ?

I think it will be nice to have it in an open issue so it is more likely to get eyeballs.